client/wc-api/settings/operations.js

```diff
@@ -0,0 +1,71 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+import { pick } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { NAMESPACE } from '../constants';
+
+function read( resourceNames, fetch = apiFetch ) {
+	return [ ...readSettings( resourceNames, fetch ) ];
+}
+
+function update( resourceNames, data, fetch = apiFetch ) {
+	return [ ...updateSettings( resourceNames, data, fetch ) ];
+}
+
+function readSettings( resourceNames, fetch ) {
+	if ( resourceNames.includes( 'settings' ) ) {
+		const url = '/wc/v3/settings/wc_admin';
```

Comment:
Should this use `NAMESPACE`?

Reply:
Hmm, yes, this should probably be bumped to v4.